libecs/include/pivot/ecs/Core/Component/description_helpers_impl.hxx

```diff
@@ -65,15 +65,18 @@ struct Helpers<T> {
 
     static void updateTypeWithValue(T &data, const data::Value &value)
     {
+        data = T{};
```

Comment:
Is it necessary to reset the whole object ? Can be expensive. Perhaps reset individual field if the value is not found

Reply:
How do you get the default value of only one property of a class/struct then ? For example for Transform I want to default value:
```c++
class Transform
{
public:
    glm::vec3 position = glm::vec3(0.0f);
    glm::vec3 rotation = glm::vec3(0.0f);
    glm::vec3 scale = glm::vec3(1.0f);
};
```